demisto_sdk/commands/common/tests/docker_test.py

```diff
@@ -132,52 +132,32 @@ def test_parse_docker_image():
 
 
 # disable-secrets-detection-end
-
-
 def test_is_docker_image_latest_tag():
```

Comment:
it is a bad practise to have 1 unit test for all cases, lets split this to a few tests which each tests a specific cases so it will be more readable